test/index.spec.js

```diff
@@ -1,13 +1,123 @@
-const chai = require('chai')
-const chaiHttp = require('chai-http')
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const Mail = require("../server/helper/mail");
 
-chai.use(chaiHttp)
-const expect = chai.expect
+chai.use(chaiHttp);
+const { expect } = chai;
+chai.should();
 
 const add = 2 + 2;
-describe('Sample Test', () => {
-    it('Add number', () => {
-       expect(add).to.eql(4)
-    })
-    
-})
+describe("Sample Test", () => {
+  it("Add number", () => {
+    expect(add).to.eql(4);
+  });
+});
+
+// tests for helper Mail class
+describe("Tests for helper Mail class", () => {
+  let mailTest;
+  let mailTestNoSubject;
+  beforeEach(() => {
+    // create a new instance of the Mail object with a subject
+    mailTest = new Mail("Email Verification");
+
+    // create a new instance of the Mail object without a subject
+    mailTestNoSubject = new Mail();
+  });
+  // sample data to use as body of the  mail
+  const data = {
+    name: "John",
+    link: "somelinktoverifyuser"
+  };
+
+  it("It should fail if no recipient is specified", () => {
```

Comment:
Remove the `It` from all test descriptions. It's very important that the tests be readable as they provide low level documentation for the project

Reply:
I have fixed this.